pudl_scrapers/items.py

```diff
@@ -15,13 +15,26 @@ class DataFile(scrapy.Item):
 
 class Eia860(DataFile):
     """The Eia860 forms in a zip file."""
+
     year = scrapy.Field(serializer=int)
 
     def __repr__(self):
         return "Eia860(year=%d, save_path='%s')" % (
             self["year"], self["save_path"])
 
 
+class Eia860M(DataFile):
+    """The Eia860M forms in a zip file."""
+
+    year = scrapy.Field(serializer=int)
+    month = scrapy.Field(serializer=str)
```

Comment:
At the end of the process when we finally save the scraped files, can we translate the month from a string to a zero-padded number in the ISO format, e.g. `eia860m-2020-08.xlsx` rather than `eia860m-2020-August.xlsx`, in order to limit the number of different kinds of parsing of filenames that we need to do on the other end (Zenodo and PUDL datastore).

Reply:
This is a good point! I changed the file name to be zero padded number version of the month.